site/content/docs/5.3/examples/cards-rtl/index.html

```diff
@@ -19,6 +19,16 @@ <h1 class="visually-hidden">أمثلة على البطاقة</h1>
     </symbol>
   </svg>
 
+  <div class="alert alert-info mb-0" role="alert">
+    <span class="alert-icon"><span class="visually-hidden">Information</span></span>
+    <div>
+      <h4 class="alert-heading">Buttons on the last card of each line</h4>
+      <p>Please make sure to use an <code>`a`</code> for buttons if they link to another page.</p>
```

Comment:
Should we add a part about the correct HTML element to use if users don't want to make a link to another page? Or do you will they will know it implicitly?

Reply:
I don't think it's necessary since it's only replacing the `button` by `a href="..."`